src/Simplex/Chat.hs

```diff
@@ -1143,17 +1148,27 @@ deliverMessage Connection {connId, agentConnId} msgBody msgId = do
 
 sendGroupMessage :: ChatMonad m => [GroupMember] -> ChatMsgEvent -> m ()
 sendGroupMessage members chatMsgEvent = do
-  let msgBody = directMessage chatMsgEvent
-      newMsg = NewMessage {direction = MDSnd, chatMsgEventType = toChatEventTag chatMsgEvent, msgBody}
-  msgId <- withStore $ \st -> createNewMessage st newMsg
-  -- TODO once scheduled delivery is implemented memberActive should be changed to memberCurrent
-  forM_ (map memberConn $ filter memberActive members) $
-    traverse (\conn -> deliverMessage conn msgBody msgId)
+  (msgId, msgBody) <- createSndMessage chatMsgEvent
+  for_ (filter memberCurrent members) $ \m@GroupMember {groupMemberId} ->
```

Comment:
here it does filter current, but not above. That's why I suggested refactoring - sending to the group would also include filtering current members.